x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_definition_section.tsx

```diff
@@ -88,25 +88,34 @@ const Filters = ({ filters, dataViewId, index, 'data-test-subj': dataTestSubj }:
 
   const flattenedFilters = mapAndFlattenFilters(filters);
 
+  const FiltersWrapper = styled.div`
+    max-width: 600px;
+  `;
```

Comment:
What if at some point we get another component on the Rule Details page that will grow in width and break the page's layout again? Can we make the layout itself robust enough so it doesn't require all child components to limit their size correctly?

Reply:
Fixed. 

You can check the description for details, but basically our layout was being broken when extremely long strings (such as filters) couldn't be broken up based on our `overflow-wrap` default value. 

Fixed it so that if any field within our rule section grows in width because of non-breakable-strings it will be properly wrapped and won't break the layout. 

If any other component grows for any other reasons, the layout resizes and deals with it as expected.